### DISCUSSION TITLE
一分钟读论文：《跨站脚本攻击22年：全面调查及系统映射》

### DISCUSSION THREAD

**Discussion by giscus** [unbug/unbug.github.io#23] [Announcements]
# 一分钟读论文：《跨站脚本攻击22年：全面调查及系统映射》

作为前端工程师你真的了解 XSS 吗？越来越多超级应用基于 Web 构建，如何系统的攻防 XSS 尤为重要。微软安全工程师 1999 年底披露了 XSS，22年来 XSS 一直稳居 OWASP（开放 Web 应用程序安全 ） 报告的前 TOP10。阿尔及利亚的拉尔比特贝西大学、盖勒马大学和巴吉莫赫塔尔大学的 LRS 实验室的论文《Twenty-two years since revealing cross-site scripting attacks: a systematic mapping and a comprehensive survey》 调查了147份高质量研究，总结制定了一个全面的分类法，描述了用于预防、检测、保护和防御 XSS 攻击的不同技术。

https://unbug.github.io/Twenty-two-years-since-revealing-cross-site-scripting-attacks/

**Comment by unbug** [comment]
SXSS DXSS WXSS CR-XSS UXSS JSM-XSS MXSS XAS CS-XSS 😂
